src/compass/utils/runconfig.py

```diff
@@ -32,7 +32,27 @@ def validate_group_dict(group_cfg: dict) -> None:
     # burst is assigned and valid (required by geo2rdr and resample)
     is_reference = input_group['reference_burst']['is_reference']
     if not is_reference:
-        helpers.check_file_path(input_group['reference_burst']['file_path'])
+        # check if reference directory exists
+        ref_path = input_group['reference_burst']['file_path']
+        helpers.check_directory(ref_path)
+
+        for burst_id in input_group['burst_id']:
+            # check if a directory exists for each burst ID
+            burst_id_path = f'{ref_path}/{burst_id}'
+            helpers.check_directory(burst_id_path)
+
+            # ensure only 1 directory under burst ID path
+            n_dir_in_burst_id_path = len(os.listdir(burst_id_path))
```

Comment:
lines 45-55 are a bug. If your `burst_id_path='t71_iw3_b845'`, `os.listdir(burst_id_path)` will list the files contained in the directory. I am not sure these checks make much sense. At the end of the day, we just want that a path with a certain burst ID exists

Reply:
I don't think this is a bug. `ref_path` should be the directory where the burst_id directories reside. This accounts for multiple burst ids in an AOI defined in the schema in `groups/input_file_group/burst_id`. The schema documentation does not reflect this and I will update it accordingly. Sample directory/file structure.
```
Mar 21, 09:18:33 lyu@RAYL-C01436:sentinel1$ tree cslc1
cslc1
├── t71_iw3_b846
│   └── 2020-05-11
│       ├── VV.hdr
│       ├── ...
│       └── z.rdr
└── t71_iw3_b847
    └── 2020-05-11
        ├── VV.hdr
        ├── ...
        └── z.rdr
```